### PR TITLE
Add instructions on how 3rd party libraries can be supplied to a Pod

### PR DESCRIPTION
**Pull request by errcode1202**

### CODE REVIEW

File: docs/docs/examples/external_libraries/EXTERNAL_LIBS.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+`JDBC` drivers.
+
+There are 2 strategies for doing this, you can either use the required prerequisite [shared home volume](../../userguide/PREREQUISITES.md#configure-a-shared-home-volume) or create
+a custom volume specifically for this purpose. Each approach will be discussed below.
+
+!!!info "Approach"
+
+    Which approach is used is totally up to you. For convenience you may want to just use [shared-home](../../userguide/PREREQUISITES.md#configure-a-shared-home-volume), or if you'd like to 
+    keep things clean you may decide to mount these 3rd party libraries in a volume of their own. This approach would be 
+    particularly usefull when these libraries need to be shared with other Pod's in your cluster.
```
Comment (nghazali) [inline docs/docs/examples/external_libraries/EXTERNAL_LIBS.md:15]: usefull => useful

File: docs/docs/userguide/PREREQUISITES.md [hunk 2]
```diff
@@ -46,19 +47,23 @@ Before installing the Data Center Helm charts you need to set up your environmen
...
+* Some cloud based options for a shared filesystem include [AWS EFS](https://aws.amazon.com/efs/){.external} and [Azure Files](https://docs.microsoft.com/en-us/azure/storage/files/storage-files-introduction){.external}. You can also stand up your own NFS
+* The logical representation of the chosen storage type within Kubernetes can be defined as `PersistentVolumes` with an associated `PersistentVolumeClaims` in a `ReadWriteMany (RWX)` access mode.
 * For more information about volumes go to the [Volumes section of the configuration guide](CONFIGURATION.md#volumes). 
+* See examples of [creating shared storage](../examples/storage/STORAGE.md).
+
+### Configure local-home volume
+* As with the [shared-home](#configure-a-shared-home-volume) each pod requires its own volume for `local-home`. This is needed by each product for defining operational data. 
+* If not defined, an [emptyDir{}](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir){.external} will be utilised. 
+* Whilst an `emptyDir` may be acceptable for evaluation purposes its recommended that each Pod is allocated its own volume
```
Comment (nghazali) [inline docs/docs/userguide/PREREQUISITES.md:68]: I am not an expert writer, but I think better use `it is recommended`

**Review by nghazali — APPROVED** [review]

**Comment by nghazali** [comment]
Just a couple of minor points.

File: docs/docs/userguide/PREREQUISITES.md (hunk 2 above)
Comment (errcode1202) [reply docs/docs/userguide/PREREQUISITES.md:68]: Not sure either actually, I do know Im missing an apostrophe. Ill update to your recommendation anyway.

File: docs/docs/examples/external_libraries/EXTERNAL_LIBS.md (hunk 1 above)
Comment (errcode1202) [reply docs/docs/examples/external_libraries/EXTERNAL_LIBS.md:15]: 🦅 👁️

**Pull request merged**